response_operations_ui/controllers/cir_controller.py

```diff
@@ -22,6 +22,16 @@ def get_cir_service_status():
     return _get_response_content(app.config["CIR_API_URL"] + "/status")
 
 
+def get_cir_metadata(survey_id, formtype):
+    # The CIR API requires both classifier_type and language to be present as part of the call in order to obtain the
+    # Relevant survey with form type. After speaking with Alina, as RASRM only deals with English surveys
+    # And as we know the classifier type will be 'form_type' it was agreed to hardcode this params
+    cir_url_query_parameters = (
```

Comment:
Could the API location be moved to either the existing `CIR_API_URL`, or a separate configuration variable rather than hardcoded in the client? As it stand it isn't one of the querystring parameters so probably shouldn't prefix them.